src/Rule/Number.php

```diff
@@ -15,12 +15,14 @@
 use Yiisoft\Validator\WhenInterface;
 
 /**
- * Validates that the value is a number.
+ * Defines validation options to check that the value is a number.
  *
  * The format of the number must match the regular expression specified in {@see Number::$integerPattern}
  * or {@see Number::$numberPattern}. Optionally, you may configure the {@see Number::min()} and {@see Number::max()}
```

Comment:
In case of such explanation, `$exactly` can be mentioned too.

Reply:
There's no `$exactly`.

Reply:
Oops, sorry, this will be actual only after merging #422.